### PR TITLE
Ux/inspector sections

### PR DESCRIPTION
**Pull request by maltenuhn**
**Problem:**
The inspector had become hard to (visually) navigate, especially with the varying length and visibility of sections / subsections.

**Fix:**
- Remove clutter by moving the right menu to the top menu
- Add icons for sections / subsections
- Create visually consistent sections / subsection headers

image

### CODE REVIEW

**Comment by Rheeseyb** [comment]
Wrong icon for the border section?
Screenshot_20210810_114327

**Review by balazsbajorics — APPROVED** [review]

File: editor/src/uuiui/icons.tsx [hunk 1]
```diff
@@ -70,6 +70,79 @@ export const LargerIcons = {
...
+  Background: makeIcon({
+    category: 'inspector',
+    type: 'background',
+    color: 'main',
+    width: 16,
+    height: 16,
+  }),
+  Border: makeIcon({
+    category: 'inspector',
+    type: 'layout',
```
Comment (Rheeseyb) [inline editor/src/uuiui/icons.tsx:104]: This would be the reason for the incorrect border icon

Comment (balazsbajorics) [reply editor/src/uuiui/icons.tsx:104]: 🧐

Comment (maltenuhn) [reply editor/src/uuiui/icons.tsx:104]: how peculiar!

**Comment by Rheeseyb** [comment]
This appears to have pushed the render count tests up quite a lot :thinking:

**Comment by maltenuhn** [comment]
> This appears to have pushed the render count tests up quite a lot 🤔

Yeah, but why? If anything, I _removed_ some things that were in the right menu before...

**Comment by Rheeseyb** [comment]
> > This appears to have pushed the render count tests up quite a lot thinking
> 
> Yeah, but why? If anything, I _removed_ some things that were in the right menu before...

At the least I can see that we weren't previously re-rendering the right menu when selection changed, whereas we are now re-rendering those buttons since moving them to the top bar, so fixing that should drop the render count. I guess the addition of the icons will push the count up a little too, but that part is fine.

**Review by Rheeseyb — APPROVED** [review]

**Pull request merged**